service/src/main/java/bio/terra/pipelines/db/entities/converters/QuotaUnitsConverter.java

```diff
@@ -0,0 +1,19 @@
+package bio.terra.pipelines.db.entities.converters;
+
+import bio.terra.pipelines.common.utils.QuotaUnitsEnum;
+import jakarta.persistence.AttributeConverter;
+import jakarta.persistence.Converter;
+
+// inspired by https://www.baeldung.com/jpa-persisting-enums-in-jpa
+@Converter(autoApply = true)
+public class QuotaUnitsConverter implements AttributeConverter<QuotaUnitsEnum, String> {
+  @Override
+  public String convertToDatabaseColumn(QuotaUnitsEnum quotaUnitsEnum) {
```

Comment:
this is the one place that's not being tested. since we never actually use code to set this value in the db (we use liquibase), i'm ok with that, but i can construct a test if that would be better (we don't have explicit tests of these kinds of classes currently)

Reply:
yea i think thats more than fine to not test this.  We could prob even throw like a "not implemented error dont call me" here and it wouldnt be an issue.  that could be the solution :P